com.writerduet.writersolo.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: com.writerduet.writersolo
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: runwritersolo
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --filesystem=home
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
```suggestion
```

This isn't needed, the baseapp comes with libnotify 0.8.3, notifications go through portal without any permissions.